### DISCUSSION TITLE
If the Kts of the aircraft exceeds 800, there will be jitter problems？

### DISCUSSION THREAD

**Discussion by JoinKomorebi** [JSBSim-Team/jsbsim#695] [Q&A]
Sometimes, when the speed of the aircraft's Kts exceeds 800, the problem of aircraft shaking will occur. I would like to ask if this is a normal phenomenon, or is there any problem, and how to avoid this problem?

**Comment by bcoconni** [comment]
Hi @JoinKomorebi,

Could you please tell us which aircraft you are talking about ? Is it an aircraft from JSBSim or your own work ?

Also it would be helpful if you could provide us with more details about what exactly is going wrong on your aircraft ? I suggest that you read the discussions #675, #629 and #444 where users have reported issues similar to yours. These discussions are also good examples of the kind of input that you can provide to feed the discussion.

**Reply by JoinKomorebi** [reply]
Thanks For your Answer
I am using the f16 model now, I refer to the link you sent me, and update the version of the JSBSim plugin, there is no problem of the plane shaking when the speed is high
